composer.json

```diff
@@ -1,6 +1,7 @@
 {
   "require": {
     "krokedil/wp-api": "^1.0",
+    "krokedil/sign-in-with-klarna": "@dev"
```

Comment:
These needs to be changed before we can merge this branch. So lets wait until we have a released version of the package